tests/test-helper.js

```diff
@@ -1,7 +1,7 @@
 import Ember from 'ember';
 import resolver from './helpers/resolver';
 import QUnit from 'qunit';
-import { setResolver } from 'ember-qunit';
+import { setResolver } from '@ember/test-helpers';
 import { start } from 'ember-cli-qunit';
 
 setResolver(resolver);
```

Comment:
Should update this to `setApplicztion`

Reply:
Is that the Russian version ;)? Also, can you help me understand this change?

Reply:
Lol, well i think it may be slavic? Will DM to chat RE: what this does

Reply:
Rgr